index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MOOC-Capstone Webpage</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="capstone.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+                
+</head>
+<body>
+    <section class="home-page">
+        <nav class="home-nav">
+            <ul>
+                <li><i class="fab fa-facebook-f"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log Out</a></li>
+            </ul>
+        </nav>
+        <header class="header">
+            <nav>
+                <i class="fas fa-bars hamburger"></i>
+            </nav>
+            <div class="desk-nav">
+                <i class="fas fa-globe">MOOC</i>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a href="#">Sponsors</a></li>
+                    <li class="call"><a href="#">Call us</a></li>
+                </ul>
+            </div>
+        </header>
+        <div class="mobile-menu">
+            <div class="time">
+                <h3>09:41</h3>
+                <ul class="icon-left">
+                    <li><i class="fas fa-signal"></i></li>
+                    <li><i class="fas fa-wifi"></i></li>
+                    <li><i class="fas fa-battery-full"></i></li>
+                </ul>
+            </div>
+            <i class="fas fa-times cancel"></i>
+            <ul class="menu-list">
+                <li><div class="box"><a href="#work">Portfolio</a></div></li>
+                <li><div class="box"><a href="#about">About</a></div></li>
+                <li><div class="box"><a href="#contact">Contact</a></div></li>
+            </ul>
+        </div> 
```

Comment:
- [ ] I think your current mobile menu is from the `portfolio` project which does not match this project's mobile menu. Your current capstone project should have a different menu that contains the links to `home `and `about `pages. Kindly fix this issue, please.
Below is your current mobile menu 👇 
![Screenshot (185)_LI](https://user-images.githubusercontent.com/67087664/153597245-10492b17-9b97-4687-9870-871de618ef01.jpg)

Reply:
Please make this change that was suggested currently your popup doesn't match the Figma in terms of colors and the `font`s used. 

![image](https://user-images.githubusercontent.com/18730106/153673055-f622f66f-91d7-4c3e-b0a4-b809aba3cdcd.png)